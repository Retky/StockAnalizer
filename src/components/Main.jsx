import React from 'react';

const Main = () => (
  <section>
    A
  </section>
);

export default Main;
