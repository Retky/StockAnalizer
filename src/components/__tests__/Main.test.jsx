import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/configureStore';
import Main from '../Main';

describe('Main component', () => {
  test('component match with the snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Main />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
