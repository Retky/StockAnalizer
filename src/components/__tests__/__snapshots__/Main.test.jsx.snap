// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component component match with the snapshot 1`] = `
<section>
  <select
    className="filter"
    onChange={[Function]}
  >
    <option
      value="price"
    >
       Price 
    </option>
    <option
      value="name"
    >
       Name 
    </option>
    <option
      value="vol"
    >
       24h% 
    </option>
  </select>
  <ul
    className="coinList"
  />
</section>
`;
